src/rust/graph-generator-lib/Cargo.toml

```diff
@@ -1,19 +1,21 @@
 [package]
 name = "graph-generator-lib"
-version = "1.0.0"
+version = "0.1.8"
 authors = ["Insanitybit <insanitybit@gmail.com>"]
 edition = "2018"
+description = "Library for writing Grapl generator services"
+license = "Apache-2.0"
 
 [dependencies]
-graph-descriptions = {package="grapl-graph-descriptions", path="../graph-descriptions", version="*"}
+graph-descriptions = {package="grapl-graph-descriptions", version="0.2.10"}
```

Comment:
Why is this version pinned now?

Reply:
We have to if we want to publish to crates.io

Reply:
I can revert? But we'd have to do this every time we publish